core/storage/user/gae_models.py

```diff
@@ -320,7 +333,7 @@ def export_data(user_id):
         """
         user_model = CompletedActivitiesModel.get(user_id, strict=False)
         if not user_model:
-            return None
+            return {}
```

Comment:
When making changes like this, please fix the docstring too. Ditto in multiple other places below.

Reply:
Done.